OrbitCore/TimerManager.cpp

```diff
@@ -34,7 +34,7 @@ TimerManager::TimerManager(bool a_IsClient)
 
   if (m_IsClient) {
     GTcpClient->Start();
-    m_ConsumerThread = new std::thread([&]() { SendTimers(); });
+    consumerThread_ = std::thread{[this]() { SendTimers(); }};
```

Comment:
also maybe add a CHECK on !joinable here...

Reply:
Well, it's called in the constructor here, right? So the state of `consumerThread_` should be pretty clear, shouldn't it?

Reply:
Ah, yes - I somehow read it as part of StartClient()